lib/chewy/railtie.rb

```diff
@@ -43,14 +43,12 @@ def migrate(*args)
     end
 
     console do |app|
-      Chewy.logger = ActiveRecord::Base.logger if defined?(ActiveRecord)
```

Comment:
I think it'd be cleaner this way:
```
Chewy.logger = defined?(ActiveRecord) ? ActiveRecord::Base.logger : Logger.new(STDERR)
```

Reply:
No need to do it, rails does the same, https://github.com/rails/rails/blob/master/railties/lib/rails/commands/console/console_command.rb#L58, just puts

Reply:
Well, the fact that they do the same doesn't change anything. If you assign a your own logger instance when AR is not available, you'll at least be able to get rid of that ugly conditional above

Reply:
I understand your point, I was thinking about this but decided to keep it simple now.